SU2_CFD/src/numerics_direct_turbulent.cpp

```diff
@@ -1256,10 +1262,27 @@ void CSourcePieceWise_TurbSST::ComputeResidual(su2double *val_residual, su2doubl
    else {
      pw = StrainMag_i*StrainMag_i - 2.0/3.0*zeta*diverg;
    }
-   pw = max(pw,0.0);
+   pw = alfa_blended*Density_i*max(pw,0.0);
+
+   /*--- Sustaining terms, if desired. Note that if the production terms are
+         larger equal than the sustaining terms, the original formulation is
+         obtained again. This is in contrast to the version in literature
+         where the sustaining terms are simply added. This latter approach could
+         lead to problems for very big values of the free-stream turbulence
+         intensity. ---*/
+
+   if ( sustaining_terms ) {
+     const su2double sust_k = beta_star*Density_i*kAmb*omegaAmb;
```

Comment:
These changes are so slight I missed them while scrolling to get to the meat of the feature.

Why not simply add a config option for USE_SST_SUSTAINING_TERMS instead of through a new turb solver type? (which forces you to change lots of booleans in the code)
Or maybe use the turb solver option but in the config post processing set a different boolean to indicate sustaining_terms and reset KindTurbSolver to SST.